### PR TITLE
Add Dawn to apps

### PR DESCRIPTION
**Pull request by noppefoxwolf**
Nightfox dawn is new iOS app for mastodon.
https://apps.apple.com/app/nightfox-dawn/id1668645019

### CODE REVIEW

**Comment by noppefoxwolf** [comment]
@renchap I have resolved conflicts at times. Can you merge this Pull Request?

**Comment by andypiper** [comment]
Hi, is this app available globally? I am not seeing it in the App Store (UK)

**Comment by noppefoxwolf** [comment]
Now, it's released in the US, JP, and a few other countries. Do I have to release globally to merge this PR?

**Comment by andypiper** [comment]
It does mean that I'm not able to try it myself 😄 I think it would be preferable to list apps that are available anywhere, otherwise people may be confused if they click on a link that appears to be broken.

**Comment by andypiper** [comment]
I've updating the Contributing.md file now to say that we prefer apps that are globally-available. If you choose to distribute more widely in the future, happy to look back at this PR. Thank you!

**Comment by noppefoxwolf** [comment]
Thank you for updating Contribute guideline.
I'll challenge to localize/release globally-available.

**Comment by noppefoxwolf** [comment]
@andypiper DAWN is now on available in global. I reopened this PR. Will you review again for me?

File: data/apps.ts [hunk 1]
```diff
@@ -136,7 +137,14 @@ export const apps: appsList = {
       icon: mammoth,
       url: "https://apps.apple.com/app/mammoth-for-mastodon/id1667573899",
       paid: false
-    }
+    },
+    {
+      released_on: "Mar 27, 2023",
+      name: "Nightfox DAWN",
```
Comment (andypiper) [inline data/apps.ts:144]: In the iOS App Store this is called "DAWN for Mastodon", please update this here to that it matches. Thanks!

**Review by andypiper — CHANGES_REQUESTED** [review]

**Review by andypiper — APPROVED** [review]

**Pull request merged**